### ISSUE TITLE
GTFS - “Too Fast Travel” 

### ISSUE THREAD

**Issue by barbeau** [CUTR-at-USF/psta-data#6]
This is related to #4, but some trips are almost entirely shown having the same arrival time for all stops – for example, the following trip starts at 8:30am for stop_sequence 1, and then jumps 9:30am for stop_sequence 2, and all remaining stops on this trip are scheduled for 9:30am:

```
trip_id,arrival_time,departure_time,stop_id,stop_sequence,stop_headsign,pickup_type,drop_off_type,shape_dist_traveled
1187493060,08:30:00,08:30:00,506,1,,0,0, 
1187493060,09:30:00,09:30:00,511,2,,0,0, 
1187493060,09:30:00,09:30:00,512,3,,0,0, 
1187493060,09:30:00,09:30:00,513,4,,0,0, 
1187493060,09:30:00,09:30:00,514,5,,0,0, 
1187493060,09:30:00,09:30:00,515,6,,0,0, 
1187493060,09:30:00,09:30:00,608,7,,0,0, 
1187493060,09:30:00,09:30:00,5520,8,,0,0, 
1187493060,09:30:00,09:30:00,610,9,,0,0, 
1187493060,09:30:00,09:30:00,611,10,,0,0,
….
```

Still exists in S0000161.

**Comment by barbeau** [comment]
CD:

> #6 “Too Fast Travel” – as you said, this is most likely related to Issue #4. I will forward this information to the data team here at Clever Devices, and we will make sure it is fixed along with the fixes for issue #4.

**Comment by barbeau** [comment]
Looks like this is fixed in `PSTA_S0000192.zip` - at least I haven't found any occurences yet, and the validator is no longer flagging it:
https://github.com/CUTR-at-USF/psta-data/issues/3#issuecomment-136755184
